Gemfile.lock

```diff
@@ -249,3 +249,6 @@ DEPENDENCIES
   uglifier (= 2.7.2)
   unicorn (~> 4.9.0)
   webmock (= 1.21.0)
+
+BUNDLED WITH
```

Comment:
It may be worth checking on integration what version of bundler our 2.3.0 installs use.  I think it might be 1.11.2 not 1.10.6, so maybe we need up this in the Bump ruby commit.